rs/nns/governance/src/neuron_store/voting_power.rs

```diff
@@ -0,0 +1,65 @@
+use crate::pb::v1::{VotingPowerMap, VotingPowerTotal};
+
+use std::collections::HashMap;
+
+/// The snapshot of voting power for all eligible neurons at a given time.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct VotingPowerSnapshot {
```

Comment:
Kind of odd that this isn't composed of VotingPowerMap and VotingPowerTotal - seems like it's the same thing?